modules/ocf_kubernetes/templates/master/loadbalancer/keepalived.conf.erb

```diff
@@ -1,23 +0,0 @@
-vrrp_instance VI_1 {
-    interface <%= @networking['primary'] %>
-    state MASTER
-    virtual_router_id 51
-    priority 100
-
-    authentication {
-        auth_type PASS
-        auth_pass $ <%= @keepalived_secret %>
-    }
```

Comment:
Why is this removed without a replacement? Just curious

Reply:
see https://github.com/ocf/puppet/pull/908#discussion_r392554803. It's deprecated and I don't think it's even doing anything right now.

Reply:
Ah I see:

```
# Note: authentication was removed from the VRRPv2 specification by
# RFC3768 in 2004.
#   Use of this option is non-compliant and can cause problems; avoid
#   using if possible, except when using unicast, where it can be helpful.
```

Well, sounds good to remove then